src/libs/NetworkConnection.js

```diff
@@ -1,7 +1,6 @@
 import _ from 'underscore';
 import NetInfo from '@react-native-community/netinfo';
 import AppStateMonitor from './AppStateMonitor';
-import promiseAllSettled from './promiseAllSettled';
```

Comment:
`promiseAllSettled` seems to be unused in the code base now. I think we should remove it as well.